internal/internal.go

```diff
@@ -188,6 +188,9 @@ var (
 	ExitIdleModeForTesting any // func(*grpc.ClientConn) error
 
 	ChannelzTurnOffForTesting func()
+
+	// Gets the function definition from metadata package
```

Comment:
Please fix formatting. We want the comment to start with the variable name. Also I feel like the comment doesnt explain what the method does, but instead talks about why this internal variable is being used. 

how about something like this? 
```suggestion
	// FromOutgoingContextRaw returns the un-merged, intermediary contents of metadata.rawMD. 
```

Reply:
thanks @arvindbr8, I've merged your commit suggestion